### ISSUE TITLE
Spring 2022 release 🌼

### ISSUE THREAD

**Issue by umarcor** [spf13/cobra#1626]
Approved by a maintainer but merge seems to have been forgotten:

- #841

Left pending in the previous releases (#1388, #1496, #1565):

- #1387 
- #1467
- #1530

Related to documentation/site rework:

- #1428

Some random cool ones:

- #1003
- #1604
- #1572
- Maybe conflicting:
  - #1231
  - #1500

See also [Milestones: Next](https://github.com/spf13/cobra/milestone/3) and [Milestones: 2.0.0](https://github.com/spf13/cobra/milestone/1).

**Comment by umarcor** [comment]
Can some maintainer please have a look at dependency/CI PRs?

- #1387
- #1615
- #1618
- #1619
- #1620
- #1635

**Comment by umarcor** [comment]
@marckhouzam can you please milestone #1551, #1625 and #1634?

**Comment by marckhouzam** [comment]
> @marckhouzam can you please milestone #1551, #1625 and #1634?

Geez 😢  I'll get it eventually 😄   Thanks for the reminder!

**Comment by umarcor** [comment]
@marckhouzam 👇🏼 

- #1615
- #1618
- #1620
- #1640

**Comment by marckhouzam** [comment]
> @marckhouzam 👇🏼

Done 😄

**Comment by PapaCharlie** [comment]
Hey team! Any updates on when this will be released? There are a couple things in this release that are going to be super handy!

**Comment by jpmcb** [comment]
Hi @PapaCharlie - thanks for checking in. A number of us are traveling, have PTO, etc. I'm hoping to cut this soon, at the latest by the end of June.

**Comment by plastikfan** [comment]
> Hey team! Any updates on when this will be released? There are a couple things in this release that are going to be super handy!

Yeah, I'm particularly looking forward to using `MarkFlagsRequiredTogether` and `MarkFlagsMutuallyExclusive`

**Comment by PapaCharlie** [comment]
@jpmcb No worries! I just have a lot of code I've been anxiously waiting to nuke once the new flag validations are added.

**Comment by c4milo** [comment]
Looking forward to this release as well! 🙌

**Comment by jpmcb** [comment]
I'm planning to cut this soon, hopefully in the next week or so!
